pymc4/model/base.py

```diff
@@ -83,9 +85,11 @@ def log_joint_fn(*args, **kwargs):  # pylint: disable=unused-argument
             states = dict(zip(self.unobserved.keys(), args))
             states.update(self.observed)
             interceptor = interceptors.CollectLogProb(states)
-            with ed.interception(interceptor):
+            temp_graph = tf.Graph()
+            tf.contrib.graph_editor.copy(self.graph, temp_graph)
+            with temp_graph.as_default(), ed.interception(interceptor):
                 self._f(self._cfg)
```

Comment:
not related to this PR, but this line should probably be `self._f(self.cfg)`, right?

Reply:
IMO it shouldnt make a difference. `self.cfg` returns `self._cfg`. Although I am not sure why we are saving it as `_cfg`